docs: add steps for contributing to docs locally (#23270)
nuxt/nuxt: docs/5.community/5.framework-contribution.md

### Set Up Documentation Website in Local Environment

> The Nuxt documentation is currently deployed within [nuxt/nuxt in `docs` folder](https://github.com/nuxt/nuxt/tree/main/docs).

Reviewer: ```suggestion
The Nuxt documentation is currently deployed within [nuxt/nuxt.com](https://github.com/nuxt/nuxt.com) as a layer.
```

Author: But actually, the url points to nowhere, the link is broken

> The Nuxt documentation is currently deployed within [nuxt/nuxt in `docs` folder](https://github.com/nuxt/nuxt/tree/main/docs).

Author: I don't find anywhere how nuxt.com website is made
